crates/uv/src/commands/project/sync.rs

```diff
@@ -87,8 +90,20 @@ pub(crate) async fn sync(
         concurrency,
     );
 
-    // TODO(konsti): Read editables from lockfile.
-    let editables = ResolvedEditables::default();
+    let editables = ResolvedEditables::resolve(
+        Vec::new(), // TODO(konsti): Read editables from lockfile
```

Comment:
Wait, but this PR doesn't actually add anything, does it?

Reply:
I intentionally removed this in favor of a default impl.

Reply:
I see, i'll revert it until we have editable from the lockfile then: https://github.com/astral-sh/uv/pull/3696